hail/python/hail/vds/variant_dataset.py

```diff
@@ -45,17 +46,60 @@ def _variants_path(base: str) -> str:
         return os.path.join(base, 'variant_data')
 
     @staticmethod
-    def from_merged_representation(mt, ref_block_fields=('GQ', 'DP', 'MIN_DP')):
+    def from_merged_representation(mt,
+                                   *,
+                                   ref_block_fields=(),
+                                   infer_ref_block_fields: bool = True):
+
+        if 'END' not in mt.entry:
+            raise ValueError(f"VariantDataset.from_merged_representation: expect field 'END' in matrix table entry")
```

Comment:
looks like these f-strings (lines 55, 58, 62, 86) without placeholders are causing `check_hail` to fail

Reply:
bleh. thanks.